model/jpa/src/main/resources/META-INF/jpa-changelog-12.0.0.xml

```diff
@@ -19,6 +19,17 @@
 
     <changeSet author="keycloak" id="map-remove-ri">
         <dropForeignKeyConstraint baseTableName="REALM_DEFAULT_GROUPS" constraintName="FK_DEF_GROUPS_GROUP"/>
+    	<createTable tableName="REALM_METADATA">
```

Comment:
Please avoid using special tables. Use attributes instead, and provide a SPI for custom extensions.